djangocms_version_locking/models.py

```diff
@@ -1,6 +1,20 @@
+from django.conf import settings
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
 
+from djangocms_versioning.models import Version
 
-class LockingModel(models.Model):
-    pass
+
+class VersionLock(models.Model):
+    created = models.DateTimeField(auto_now_add=True)
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.PROTECT,
+        verbose_name=_('locked by')
+    )
+    version = models.ForeignKey(
```

Comment:
Do you think this could be a one to one field? Think there will be at most 1 lock to a version

Reply:
It could be as the version is either locked or it's not. I guess it would depend if we wanted to save state changes as a many. I'll check.

Reply:
Just checked, It's a one to one. You're right.